### ISSUE TITLE
Accept transfer of 'eleventy' npm namespace 

### ISSUE THREAD

**Issue by jokroese** [11ty/eleventy#2460]
**Is your feature request related to a problem? Please describe.**
I am new to the Eleventy project and I feel that the need to use `@11ty/eleventy` instead of `eleventy` in commands has a negative impact on onboarding beginners. (I would guess that it is also the tiniest bit annoying for novice users too.)

Given that the repository is 1) self-described as 'useless' and 2) has ~400 weekly downloads (20,000 a year!), I suspect many beginners have issues with installing the wrong software.

**Describe the solution you'd like**
The owner of the repository has signalled an openness to transfer the `eleventy` npm namespace to this project (source: https://github.com/102/eleventy/issues/170). I think we should engage with the offer, attain the namespace and change all the documentation from `@11ty/eleventy` to `eleventy`.

**Additional context**
I am not familiar with how this would work at an npm level. However, for backwards compatibility, we should ensure that either 1) npm sets up a redirect or 2) we maintain a mirror of eleventy at `@11ty/eleventy`.

**Comment by zachleat** [comment]
No harm to cc @102 here I guess!

**Comment by joekroese** [comment]
Paraphrasing @102, my understanding is that the next steps are for the @11ty community to answer two questions:

1. Would @11ty use the `eleventy` npm namespace?
1. If so, how would a transfer work in practice?

**Comment by joekroese** [comment]
Focusing first on question 1, my intuition is that supporting the `eleventy` npm namespace would be useful (for onboarding, documentation, reduced cognitive load, branding etc.). However, I am new to the project and unfamiliar with any relevant history and concerns.

Do any of the longer-term contributors to the @11ty project have thoughts?

**Comment by zachleat** [comment]
Yeah, I absolutely think it’d be useful to have control of this—at the very least for security reasons with accidental installs

**Comment by zachleat** [comment]
Closing this one as stale.

**Comment by jokroese** [comment]
Just circling back on this since I saw the issue was closed. Reviewing where things left off, it looks like we’re close—just a few loose ends to tie up.

We agreed that having control over the `eleventy` npm namespace would be useful—for onboarding, security, and generally reducing confusion. The open question was how to make the transfer happen.

### Recap
- [x] @102 (current owner of the `eleventy` npm namespace) has said they’re open to transferring it, as long as it’s done safely.  
  [See the relevant thread here.](https://github.com/102/eleventy/issues/170)
- [x] @zachleat confirmed it would be useful for the project.  
- [ ] What we never got to was a concrete plan for the handover.

### Proposed Steps
1. **Check in with @102**  
   Confirm that they are still open to proceeding (I see they were the first to react to closing the issue so I guess so!).
1. **Confirm with Zach (@zachleat)**  
   If bandwidth was the blocker, totally understandable. If you’re happy to move ahead and just need someone else to handle the logistics, I’m happy to coordinate.
1. **Transfer the npm package**  
   - @102 makes @zachleat a maintainer of eleventy, either through the GUI or by running:  
     ```
     npm access grant maintainer zachleat eleventy
     ```
   - once this is setup, @102 can remove themselves as maintainer whenever they feel comfortable.  
   - [npm docs on package transfer](https://docs.npmjs.com/transferring-a-package-from-a-user-account-to-another-user-account) for reference.
1. **Publish the `eleventy` package**  
   - Same releases as `@11ty/eleventy`.  
   - Keep `@11ty/eleventy` intact for backwards compatibility.
1. **Communicate the change**  
   - Docs, install instructions, maybe a quick note in a release or blog post.

---

If time or logistics are the main blockers, I can help move this forward. This seems like a small but meaningful improvement that’s pretty close to done.

Would love to get this over the line if you’re up for it! 👀

**Comment by jokroese** [comment]
To start with this process: @zachleat, if you’re up for moving forward, I’m happy to handle the coordination with @102 and sort out the details.

Just need a 👍 from you!

**Comment by zachleat** [comment]
Happy to reopen if this is still on the table! I don’t think I would double-publish to both packages, but I think there’s probably a cleaner solution in code via imports from `@11ty/`

**Comment by jokroese** [comment]
Great, sounds good! That makes sense - the simplest approach is to keep `@11ty/eleventy` as the source of truth and publish `eleventy` as a thin wrapper that imports it. We can figure out the details of the implementation after the transfer.

**Comment by jokroese** [comment]
@102 – hope you're doing well! Just wanted to check in and see if you're still happy to transfer the `eleventy` npm package? If so, it would be great if you could add @zachleat as a maintainer.

Really appreciate your help with this!

**Comment by 102** [comment]
> [@102](https://github.com/102) – hope you're doing well! Just wanted to check in and see if you're still happy to transfer the `eleventy` npm package? If so, it would be great if you could add [@zachleat](https://github.com/zachleat) as a maintainer.
> 
> Really appreciate your help with this!

I have sent an invite for https://www.npmjs.com/~zachleat to be maintainer of [eleventy](https://www.npmjs.com/package/eleventy)

**Comment by 102** [comment]
@zachleat JFYI, I have removed myself from the maintainers list of [eleventy](https://www.npmjs.com/package/eleventy) package. Feel free to update the package to mirror to the `@11ty/eleventy`

**Comment by zachleat** [comment]
I wasn’t able to find Mastodon or Bluesky accounts for y’all but I did post some thank yous for this one here:

- https://fosstodon.org/@eleventy/114338577745020018
- https://bsky.app/profile/11ty.dev/post/3lmsl22cgz22t

Let me know if you have them and I’ll post edits/updates!

**Comment by zachleat** [comment]
`eleventy` v2.0.0 just shipped with a helpful error message that links back here! Thank you all again!

https://github.com/11ty/npm-bare-error
https://www.npmjs.com/package/eleventy
